study/test/src/org/labkey/test/tests/study/StudyDatasetDefaultView.java

```diff
@@ -0,0 +1,117 @@
+package org.labkey.test.tests.study;
+
+import org.jetbrains.annotations.Nullable;
+import org.junit.BeforeClass;
+import org.junit.Test;
+import org.junit.experimental.categories.Category;
+import org.labkey.test.BaseWebDriverTest;
+import org.labkey.test.Locator;
+import org.labkey.test.TestFileUtils;
+import org.labkey.test.categories.DailyA;
+import org.labkey.test.components.CustomizeView;
+import org.labkey.test.components.study.ViewPreferencesPage;
+import org.labkey.test.util.DataRegionTable;
+import org.labkey.test.util.PortalHelper;
+
+import java.util.Arrays;
+import java.util.List;
+
+@Category({DailyA.class})
+@BaseWebDriverTest.ClassTimeout(minutes = 3)
+public class StudyDatasetDefaultView extends BaseWebDriverTest
```

Comment:
Test class names should end with "Test"
```suggestion
public class StudyDatasetDefaultViewTest extends BaseWebDriverTest
```